application/helpers/expressions/em_manager_helper.php

```diff
@@ -6770,7 +6790,7 @@ public static function StartProcessingGroup($gseq = null, $anonymized = false, $
         $LEM->groupRelevanceInfo = [];
         if (!is_null($gseq)) {
             $LEM->currentGroupSeq = $gseq;
-
+            self::InitGroupRelevanceInfo();
```

Comment:
Is it correct to add this here taking into accoutn the StartProcessingGroup may have added asomething already? (haven't looked into much detail really)

Reply:
You have this line https://github.com/Shnoulle/LimeSurvey-Shnoulle/blob/1b3fa0255eaf8ba1c5d75dc1e06c69e490a69423/application/helpers/expressions/em_manager_helper.php#L4761 each time i set it agian.

And more : you NEED some not related to question id : group description are used a lot with Expresssion.

Reply:
Sorry, not that into EM to follow